src/quest_manager/views.py

```diff
@@ -387,6 +387,17 @@ def get_minutes_to_complete_list(quest):
     return minutes_list
 
 
+class QuestListViewTabTypes:
```

Comment:
We have:
ProfileViewTypes
QuestListViewTabTypes
ApprovalsTabTypes

To be more consistent in the naming, let's make it `ApprovalsViewTabTypes` <-- add the View, that adds some good context when you are seeing that code in the model.

Reply:
done